SU2_CFD/include/solvers/CFVMFlowSolverBase.inl

```diff
@@ -429,6 +437,12 @@ void CFVMFlowSolverBase<V, R>::Viscous_Residual_impl(unsigned long iEdge, CGeome
     numerics->SetTurbKineticEnergy(turbNodes->GetSolution(iPoint,0),
                                    turbNodes->GetSolution(jPoint,0));
 
+  /*--- Wall shear stress values (wall functions) ---*/
+
+  numerics->SetTauWall(nodes->GetTauWall(iPoint),
+                       nodes->GetTauWall(jPoint));
```

Comment:
Please add or modify some test cases (comp and incomp) I will need to modify the vectorized numerics to support this and it would be helpful to have a good reference.

Reply:
OK, I have the flat plate test case for this (Schubauer & Klebanoff), will add this